lib/rules/nesting-depth.js

```diff
@@ -2,6 +2,15 @@
 
 var helpers = require('../helpers');
 
+var lodash = require('lodash');
+
+var compiledCSSDoesNotCreateDepth = function (selector) {
+  return lodash.isArray(selector.content) &&
```

Comment:
Ok I'm just using the native `isArray` method now since I can always expect that to be available in a node environment